projects/js-packages/charts/src/types.ts

```diff
@@ -212,6 +212,14 @@ export type BaseChartProps< T = DataPoint | DataPointDate > = {
 	 * Legend shape
 	 */
 	legendShape?: LegendShape< T, number >;
+	/**
+	 * Legend horizontal alignment
+	 */
+	legendAlign?: 'left' | 'center' | 'right';
+	/**
+	 * Legend vertical alignment
+	 */
+	legendVerticalAlign?: 'top' | 'bottom';
```

Comment:
This would be a breaking change, so maybe now is not the time to do it, but I wonder if we might consider grouping these under a `legendOptions` prop in future, as the list is growing steadily. For now though I think maybe we should name these consistently:
```suggestion
	/**
	 * Legend horizontal alignment
	 */
	legendAlignmentHorizontal?: 'left' | 'center' | 'right';
	/**
	 * Legend vertical alignment
	 */
	legendAlignmentVertical?: 'top' | 'bottom';
```